### ISSUE TITLE
u-boot seems to do a clean build every time

### ISSUE THREAD

**Issue by abliss** [khadas/fenix#22]
This makes it hard to iterate.

Steps:
1. run "make". Let it get through building uboot, but fail somewhere later.
2. edit the makefiles to try to fix the problem.
3. run "make" again.

EXPECTED: Since u-boot built successfully, nothing should be done
ACTUAL: u-boot is rebuilt from scratch.

**Comment by numbqq** [comment]
Yes, you are right. U-boot will be rebuilt everytime. The reason why I do this is that if you do some modifications under `board/khadas/kvim2/firmware` you have to `make clean` and rebuild the u-boot or the modifications will not be compiled. And the build of u-boot is quickly, so it's OK.

Thanks.
